test/run.bats

```diff
@@ -358,6 +358,17 @@ EOF
     done
 }
 
+@test 'ch-validate succeeds on valid image' {
+   ch-validate $CHTEST_IMG
+}
+
+@test 'ch-validate fails on invalid image' {
```

Comment:
Suggest combine this with the `ch-run` missing stuff test to re-use bad image fixture code?

Reply:
Might make sense.  The full test would need both patches merged to work though.